python/lsst/dax/imgserv/getbutler.py

```diff
@@ -0,0 +1,70 @@
+#
+# LSST Data Management System
+# Copyright 2015 LSST/AURA.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+#
+# This code is used to to select an image or a cutout of an image
+# that has its center closest to the specified RA and Dec. The
+# image is retrieved using the Data Butler.
+
+"""
+This module is used to instantiate or fetch from cache the
+appropriate butler instance.
+
+@author: John Gates, SLAC
+@author: Brian Van Klaveren, SLAC
+@author: Kenny Lo, SLAC
+
+"""
+from flask import current_app, has_app_context
+import lsst.daf.persistence as dafPersist
+
+class ButlerGet:
```

Comment:
The style guide says to name the `.py` file after the class that it contains.  https://developer.lsst.io/coding/python_style_guide.html#a-python-source-file-name-should-be-camelcase-with-leading-lowercase-and-ending-in-py